__tests__/partials.ts

```diff
@@ -15,7 +15,7 @@ test('it allows for injecting an HTML partial', async () => {
     },
   });
   const result = await build(temp.dir, {
-    partialDirectory: temp.path('partials'),
+    partialDirectory: [temp.path('partials')],
```

Comment:
Would you mind putting these test cases back they way they were before? They shouldn't need to change.